Can't install @huntersofbook/plausible-nuxt with NPM

Node v16.16.0

I tried installing `@huntersofbook/plausible-nuxt` with

```bash
npm i -D @huntersofbook/plausible-nuxt
```

or

```bash
npm add @huntersofbook/plausible-nuxt
```

but always receive the following error message:

```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

The installation of other NPM packages works without any problems.

--- a/packages/plausible-nuxt/package.json
+++ b/packages/plausible-nuxt/package.json
@@ -46,7 +46,7 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "@huntersofbook/plausible-vue": "workspace:*",
+    "@huntersofbook/plausible-vue": "^0.4.3",
     "@nuxt/kit": "^3.0.0",
     "defu": "^6.1.1"
   },

